### PR TITLE
Fix for issues #120, #118 and #122

### PR DESCRIPTION
**Pull request by simonech**

### CODE REVIEW

**Comment by simonech** [comment]
Hey @Shazwazza , remember to merge this pull request :)

**Comment by Shazwazza** [comment]
Yup!! :) I'll do this today

File: src/Articulate/BlogMlImporter.cs [hunk 1]
```diff
@@ -219,7 +220,19 @@ private BlogMLDocument GetDocument(string fileName)
...
+                IContent postNode;
+
+                //Use post.id if it's there
+                if (!String.IsNullOrWhiteSpace(post.Id))
+                {
+                    postNode = allPostNodes.FirstOrDefault(x => x.GetValue<string>("importId") == post.Id);                    
+                }
+                else //Use the "slug" (post name) if post.id is not there
+                {
+                    postNode = allPostNodes.FirstOrDefault(x => x.GetValue<string>("umbracoUrlName") != null && x.GetValue<string>("umbracoUrlName").StartsWith(post.Name.Content));
```
Comment (Shazwazza) [inline src/Articulate/BlogMlImporter.cs]: Should we make this an invariant string compare ?

Comment (simonech) [inline src/Articulate/BlogMlImporter.cs]: Yeah, might be a good idea just to be safe, even if, in theory the `umbracoUrlName` should be copied from the content slug

**Pull request merged**